### DISCUSSION TITLE
Electrode SOH Plotting

### DISCUSSION THREAD

**Discussion by wass-cloud** [pybamm-team/PyBaMM#3061] [General]
Is there a better way to plot the SOH? The example provided yields 2 plots for the positive electrode and negative electrode, but how can PyBaMM be used to convert those values into a SOH graph more commonly seen in literature where SOH is shown as a percentage with respect to time?

**Comment by valentinsulzer** [comment]
There's lots of ways to define "SOH". The most common way is "capacity divided by initial capacity". You can access capacity as either the theoretical capacity (infinitely low C-rate) `solution.summary_variables["Capacity [A.h]"]` or the actual measured capacity from Coulomb counting the simulated experiment `solution.summary_variables["Measured capacity [A.h]"]`.

**Reply by wass-cloud** [reply]
I see that makes sense with what I know for SoC, where I planned on using the Coloumb counting method on my simulated model in a similar manner to the examples. If SoH and SoC can be used basically interchangeably, then what does the ElectrodeSOH solver class do? Is it just a way to determine the amount of active material within a positive or a negative electrode?
